### DISCUSSION TITLE
Deprecation notices in every console command following 5.4 upgrade

### DISCUSSION THREAD

**Discussion by crmpicco** [symfony/symfony#49167] [Q&A] [answered]
I am quite confused by the following deprecation notice in my 5.4 application.

`The "Symfony\Component\Console\Command\Command::$defaultName" property is considered final. You should not override it in "CRMPiccoBundle\Console\Command\Aws\Cognito\CreateUser".`

In my code I have the following:
```php
class CreateUser extends Command
{
    protected static $defaultName = 'crmpiccobundle:aws:cognito:createuser';
```
...which aligns with [the documentation](https://symfony.com/doc/5.4/console.html#creating-a-command).

The parent `Command` class looks like this:
```php
    /**
     * @var string|null The default command name
     */
    protected static $defaultName;
```
All my commands are now outputting a deprecation notice, which is not ideal.
```
composer show | grep console
symfony/console                      v5.4.19            Eases the creation of beautiful and testable command line interfaces
```
What am I misunderstanding here?

**Comment by ahundiak** [comment, marked as answer]
Just in case anyone is wondering the poster has somehow managed to upgrade `symfony/error-handler` to version 6 in spite of having a 5.4 app.  Downgrading to v5 fixed the problem.
